style.css

```
@@ -2005,6 +2005,19 @@ body[data-bg="wallpaper"] .searchbar {
 }
 
+.searchbar.hiddenByOption {
+	display: none;
+	opacity: 1;
+}
+
+@media screen and (max-width: 500px) {
+	.searchbar.hiddenByOption {
+		display: block;
+
+		opacity: 0;
```

Stylelint (17.2.0)

[error] Unexpected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)